app/containers/Pub/PubVersions.js

```diff
@@ -67,6 +306,15 @@ export const PubVersions = React.createClass({
 		const isLoading = this.props.isLoading;
 		const errorMessage = this.props.error;
 		const versions = this.props.versionsData || [];
+		const pdftexTemplates = this.state.pdftexTemplates || {};
+		const selectedTemplate = this.state.selectedTemplate || 'default';
+		const selectedTemplateMetadata = (pdftexTemplates && pdftexTemplates[selectedTemplate]) ? pdftexTemplates[selectedTemplate].metadata : {};
+		const missingMetadata = this.state.missingMetadata || [];
+		const metadata = this.state.metadata;
+		const outputTypes = this.state.exportOutputTypes || [];
+
+
+		window.pdftexTemplates = pdftexTemplates;
```

Comment:
What's the logic behind assigning this to window? So we don't request it again? Or is there something else?